README.md

```diff
@@ -9,117 +19,62 @@ This repository explores the application of XL-Net on user review based question
 - **static** : contains image for the frontend
 
 ## Setup
-Clone repository and update python path
-```
-repo_name=Insight_Project_Framework # URL of your new repository
-username=mrubash1 # Username for your personal github account
-git clone https://github.com/$username/$repo_name
-cd $repo_name
-echo "export $repo_name=${PWD}" >> ~/.bash_profile
-echo "export PYTHONPATH=$repo_name/src:${PYTHONPATH}" >> ~/.bash_profile
-source ~/.bash_profile
-```
-Create new development branch and switch onto it
-```
-branch_name=dev-readme_requisites-20180905 # Name of development branch, of the form 'dev-feature_name-date_of_creation'}}
-git checkout -b $branch_name
-```
 
-## Initial Commit
-Lets start with a blank slate: remove `.git` and re initialize the repo
-```
-cd $repo_name
-rm -rf .git   
-git init   
-git status
-```  
-You'll see a list of file, these are files that git doesn't recognize. At this point, feel free to change the directory names to match your project. i.e. change the parent directory Insight_Project_Framework and the project directory Insight_Project_Framework:
-Now commit these:
+#### Installation
 ```
-git add .
-git commit -m "Initial commit"
-git push origin $branch_name
+git clone https://github.com/hairong-wang/XLNet_learn2learn.git
+cd XLNet_learn2learn
 ```
+#### Requisites
+tensorflow-gpu==1.15.0-rc1
+absl-py==0.8.0
+Flask==1.1.1
+pip
 
-## Requisites
+#### Environment setup
+Everything needed fot the environment
 
-- List all packages and software needed to build the environment
-- This could include cloud command line tools (i.e. gsutil), package managers (i.e. conda), etc.
+## Steps to run
 
-#### Dependencies
+### Step1: Configuration 
 
```

Comment:
Need to add details here.

Reply:
Will open a new branch specifically for readme, and add details in that branch.